Tasks/XamariniOS/task.json

```diff
@@ -47,6 +47,30 @@
             "helpMarkDown": "Standard configurations are Ad-Hoc, AppStore, Debug, Release."
         },
         {
+            "name": "clean",
+            "type": "boolean",
+            "label": "Clean",
+            "defaultValue": "false",
+            "required": false,
+            "helpMarkDown": "Run a clean build (/t:clean) prior to the build."
```

Comment:
We don't default to "true" in the msbuild task as well since it will increase the build time. It is a convenience option for customers to choose if they wish. 
Prior to cycle 9 /t:Clean will run but won't clean up everything (IPAs and MSYMs), rest of the output will be cleaned up. In Cycle9 Xamarin stopped putting IPAs in timestamped folders and /t:Clean will remove all build output.